### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by Kbrkannan55**
avatar2
`hi`
```swift
var name: String = "Hello World"
```

### CODE REVIEW

File: Login.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+1234567890
+This is the login page
```
Comment (Kbrkannan55) [inline Login.txt:2]: Good

File: Login.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+1234567890
```
Comment (mathanraj0601) [inline Login.txt:1]: okay

Comment (Kombaiya-MK) [inline Login.txt:1]: okay

**Review by Kombaiya-MK — APPROVED** [review]

**Review by mathanraj0601 — COMMENTED** [review]
Okay boopathi nicce work

**Review by mathanraj0601 — APPROVED** [review]
Okay boopathi nice work

**Review by EzhilAdhavan — APPROVED** [review]
Approved good job.

**Pull request merged**